.eslintrc-js.js

```diff
@@ -0,0 +1,153 @@
+          pascalCase: true
+        }
+      }
+    ],
+    'unicorn/import-index': 'error',
+    'unicorn/import-style': 'error',
+    'unicorn/new-for-builtins': 'error',
+    'unicorn/no-abusive-eslint-disable': 'error',
+    'unicorn/no-array-callback-reference': 'error',
+    'unicorn/no-array-for-each': 'off',
+    'unicorn/no-array-push-push': 'error',
+    'unicorn/no-array-reduce': 'error',
+    'unicorn/no-console-spaces': 'error',
+    'unicorn/no-for-loop': 'error',
+    'unicorn/no-hex-escape': 'error',
+    'unicorn/no-instanceof-array': 'error',
+    'unicorn/no-keyword-prefix': 'error',
+    'unicorn/no-lonely-if': 'error',
+    'unicorn/no-nested-ternary': 'error',
+    'unicorn/no-new-array': 'error',
+    'unicorn/no-new-buffer': 'error',
+    'unicorn/no-null': 'error',
+    'unicorn/no-object-as-default-parameter': 'error',
+    'unicorn/no-process-exit': 'off',
+    'unicorn/no-this-assignment': 'error',
+    'unicorn/no-unreadable-array-destructuring': 'error',
+    'unicorn/no-unsafe-regex': 'error',
+    'unicorn/no-unused-properties': 'error',
+    'unicorn/no-useless-undefined': 'error',
+    'unicorn/no-zero-fractions': 'error',
+    'unicorn/number-literal-case': 'error',
+    'unicorn/numeric-separators-style': 'error',
+    'unicorn/prefer-add-event-listener': 'error',
+    'unicorn/prefer-array-find': 'error',
+    'unicorn/prefer-array-flat-map': 'error',
+    'unicorn/prefer-array-index-of': 'error',
+    'unicorn/prefer-array-some': 'error',
+    'unicorn/prefer-date-now': 'error',
+    'unicorn/prefer-default-parameters': 'error',
+    'unicorn/prefer-dom-node-append': 'error',
+    'unicorn/prefer-dom-node-dataset': 'error',
+    'unicorn/prefer-dom-node-remove': 'error',
+    'unicorn/prefer-dom-node-text-content': 'error',
+    'unicorn/prefer-includes': 'error',
+    'unicorn/prefer-keyboard-event-key': 'error',
+    'unicorn/prefer-math-trunc': 'error',
+    'unicorn/prefer-modern-dom-apis': 'error',
+    'unicorn/prefer-negative-index': 'error',
+    'unicorn/prefer-number-properties': 'error',
+    'unicorn/prefer-optional-catch-binding': 'error',
+    'unicorn/prefer-query-selector': 'error',
+    'unicorn/prefer-reflect-apply': 'error',
+    'unicorn/prefer-regexp-test': 'error',
+    'unicorn/prefer-set-has': 'error',
+    'unicorn/prefer-spread': 'error',
+    'unicorn/prefer-string-replace-all': 'error',
+    'unicorn/prefer-string-slice': 'error',
+    'unicorn/prefer-string-starts-ends-with': 'error',
+    'unicorn/prefer-string-trim-start-end': 'error',
+    'unicorn/prefer-ternary': 'error',
+    'unicorn/prefer-type-error': 'error',
+    'unicorn/prevent-abbreviations': 'off',
+    'unicorn/string-content': 'error',
+    'unicorn/throw-new-error': 'error',
+    'unused-imports/no-unused-imports-ts': 'error',
+    'woke/LGBTQ': 'error',
+    'woke/gender': 'error',
+    'woke/profanity': 'error',
+    'woke/racism': 'off' // need to find another abbreviation for "production"
```

Comment:
This could be interpreted badly... "we turned our racism filters off"!

Reply:
To fix this we need to update our vocabulary. We're currently in violation because we use the word `prod` (which I discovered is an offensive term).

Reply:
This is now fixed!